server/src/settings.py

```diff
@@ -6,8 +6,9 @@ class Settings(BaseSettings):
     # not used atm. was meant for the filter tasks which were set up on prod
     #prod_api_key: str
     analytics_api_url: str
-    tenant_id: str
+    tenant_id: int
```

Comment:
why did this work before? bitmark api expects an int not a string. i checked, a string was sent over. seems bitmark api is quite forgiving.

Reply:
guess the converter is not that strict and tries to base 10 parse it